src/script/calling/CallingRepository.ts

```diff
@@ -1666,13 +1679,43 @@ export class CallingRepository {
       userId: this.parseQualifiedId(member.userid),
     }));
 
+    this.handleOneToOneMlsCallParticipantLeave(conversationId, members);
     this.updateParticipantList(call, members);
     this.updateParticipantMutedState(call, members);
     this.updateParticipantVideoState(call, members);
     this.updateParticipantAudioState(call, members);
     this.handleCallParticipantChange(conversationId, members);
   };
 
+  private readonly handleOneToOneMlsCallParticipantLeave = (
+    conversationId: QualifiedId,
+    members: QualifiedWcallMember[],
+  ) => {
+    const conversation = this.getConversationById(conversationId);
+    const call = this.findCall(conversationId);
+
+    const [, nextOtherMember] = members;
```

Comment:
Are we sure that the next member will always be the other person who left the call?
I am not sure how members are placed in array? Maybe the first one will be the one who started the call.
Is it not better if we find the other using current userId, for example:
`const nextOtherMember = members.find(member => member.id !== -currentUserId-)`

Reply:
I agree with Immad. Even if the order is guaranteed right now, there is no guarantee that someone might change some code and, as a side effect, breaks this part. 
Searching for the right value should be a better approach.

Reply:
Yes, seems we overlooked this, we check that the id hasn't changed, but not that it's the expected one in the first place :sweat_smile: 
Let me take a look